package.json

```diff
@@ -6,11 +6,25 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "host": "vite --host"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/material": "^5.11.3",
+    "@reduxjs/toolkit": "^1.9.1",
+    "@types/react-redux": "^7.1.25",
+    "@types/styled-components": "^5.1.26",
+    "moment": "^2.29.4",
```

Comment:
There are better libraries out there that are much smaller in size. Like 
- date-fns
- Luxon
- day.js etc. 

Check if apexcharts supports one of them and use that instead of moment.js.

Reply:
Removed moment library and used "date-fns" library for date formatting and handling in the latest fix commit